src/consts.ts

```diff
@@ -30,14 +30,16 @@ const Consts = {
   },
   regexes: {
     impossible: /(?=a)b/,
-    todo: /^[^\S\n]*((?!--|––|——)(?:[-❍❑■⬜□☐▪▫–—≡→›✘xX✔✓☑+]|\[[ xX+-]\])[^\n]*)/,
-    todoToken: /^[^\S\n]*(?!--|––|——)([-❍❑■⬜□☐▪▫–—≡→›✘xX✔✓☑+]|\[[ xX+-]\])/,
-    todoBox: /^[^\S\n]*((?!--|––|——)(?:[-❍❑■⬜□☐▪▫–—≡→›]|\[ \])[^\n]*)/,
+    // todo, todoToken, and todoBox expression changed to allow for no space, e.g. []
+    todo: /^[^\S\n]*((?!--|––|——)(?:[-❍❑■⬜□☐▪▫–—≡→›✘xX✔✓☑+]|\[[ xX+-]?\])[^\n]*)/,
+    todoToken: /^[^\S\n]*(?!--|––|——)([-❍❑■⬜□☐▪▫–—≡→›✘xX✔✓☑+]|\[[ xX+-]?\])/,
+    todoBox: /^[^\S\n]*((?!--|––|——)(?:[-❍❑■⬜□☐▪▫–—≡→›]|\[ {0,1}\])[^\n]*)/,
```

Comment:
can you do ` ?` instead of ` {0,1}`? For consistency.

Reply:
Done. These RegEx gave me a lot more trouble than they should have. It seemed like I tried what I ultimately came up with and couldn't get it to work. Now it does and is consistent.